src/lib/holocene/split-button.svelte

```diff
@@ -26,7 +26,7 @@
       {label}
     </Button>
     <MenuButton
-      class="max-w-fit grow-0 px-1"
```

Comment:
I'm pretty sure this was intentional based on feedback from Ash in https://github.com/temporalio/ui/pull/1926

I'd double check with them that this is the padding we want. Looks like the designs specify p-2
<img width="527" alt="Screenshot 2024-04-30 at 9 37 01 AM" src="https://github.com/temporalio/ui/assets/11775628/2e5af22f-4b69-4880-996c-7e913336e46c">

Reply:
That was before the icons were changed to 16x16, I think px-3 is the way to go now.

<img width="587" alt="Screen Shot 2024-04-30 at 10 49 13 AM" src="https://github.com/temporalio/ui/assets/7967403/8de8848b-768f-41a8-8e22-7d816ba65dbb">